apis/tempo/v1alpha1/microservices_types.go

```diff
@@ -128,6 +134,21 @@ type Resources struct {
 	Total *corev1.ResourceRequirements `json:"total,omitempty"`
 }
 
+type SearchSpec struct {
+	// +optional
+	// +kubebuilder:validation:Optional
+	// +operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Max search time range allowed"
+	MaxSearchDuration string `json:"maxSearchDuration,omitempty"`
```

Comment:
What is the default value?

Does 0 have a special meaning? 

Is this a lookback, meaning that the range can be always from the current time or any range in the past?

Reply:
The default value is 0, which means unlimited, perhaps we should set it to the retention period?